recursive_count_th/count_th.py

```diff
@@ -4,7 +4,17 @@
 Your function must utilize recursion. It cannot contain any loops.
 '''
 def count_th(word):
-    
-    # TBC
-    
-    pass
\ No newline at end of file
+    if 'th' in word:
+        count = 1
+        index = word.find('th') + 2
+        remaining_word = word[index:]
+        return count + count_th(remaining_word)
+    else:
+        count = 0
+    return count
+
+
```

Comment:
I've not seen Python find method! Awesome! Looks great and passes all test cases.